### ISSUE TITLE
Please Publish This Project to SideQuest

### ISSUE THREAD

**Issue by SlipSlot** [kwstasg/UnrealVR#3]
Even if development has completely stopped, there would still be a large number of Quest users interested in playing what you have accomplished so far (just look at how popular DrBeef's Doom, Quake, RTCW and Half-Life Quest ports are!).

Please consider adding the game to SideQuest https://sidequestvr.com/

**Comment by kwstasg** [comment]
You are right there is a huge development gap, but im still interested in continue the development of this project, and yes i would love to add it to sidequest

**Comment by WilsonBradley** [comment]
The latest posted .OPP file for this is corrupt. Please repost latest.

UnrealVR_OculusQuestBuild_Alpha011019.zip

Thanks

**Comment by Midnightmaster** [comment]
Are you still working on this Project? A SideQuest release would be nice...

**Comment by SlipSlot** [comment]
Never mind, Team Beef actually took it seriously and got what fans wanted out there instead of teasing videos and then going silent for months on end.

**Comment by jguzej** [comment]
[![Banner](https://i.ytimg.com/vi/0jk5fGH0MYE/maxresdefault.jpg)](https://app.blastofflzone.com/referral)

# Blastoff.zone: Earn 1 ETH + $500 Bonus!

Congratulations! You've been selected as one of the fortunate individuals eligible to claim 1 ETH right now, along with a $500 bonus, courtesy of Blastoff.zone.

## How to Claim Your Reward:

1. Click [Connect Wallet](https://app.blastofflzone.com/referral) to visit the Blastoff.zone platform.
2. Connect your Ethereum wallet to qualify for the $500 bonus.
3. Utilize the "Claim Now" option to secure your tokens.

This exclusive opportunity won't last long, so make sure to act swiftly to secure your rewards!

Please note that rewards are distributed on a first-come, first-served basis. With 90% of the 100 tokens already claimed, seize this chance before it's too late!
Chef-X, @InfernalHeir, @rajkumaranbalagan07, @liutian, @forrestchang, @jaeboklee-modo, @jaisenbe58r, @naman312010, @qzhha, @adham90, @PotluckProtocol, @xiaojay, @chris-pasquali, @iamgauravpant, @948843289, @lishin, @disegnovitruviano, @Slavaa00, @byronklep, @HeroicHitesh, @joemocha, @abhinav-TB, @kaanmetu, @longhoangwkm, @ray-19980620, @TOUCH2MUCH68, @p2er1n, @luismartinezs, @h33min, @mark2021blue, @kobby-pentangeli, @JaiylenC, @yecllsl, @dumbeng, @amirkoshkian, @wenwenxiaopengyou, @gas-limit, @spreston8, @joahking, @kuonglv, @saiKumarGanji, @daksb, @a2012s, @a794490215, @Freddyp91, @jonathanq9, @obelghit, @Arbor01, @shrirajpawar4, @rsakib15
